bpf/Makefile

```diff
@@ -27,22 +27,22 @@ check:
 	clang ${CLANG_FLAGS} --analyze *.c
 
 LB_OPTIONS = \
-	-DUNKNOWN \
 	-DLB_L3 \
 	-DLB_L4 \
 	-DLB_L3 -DLB_L4
 
 bpf_lb.o: bpf_lb.c $(LIB)
 	$(foreach OPTS,$(LB_OPTIONS), \
 		${CLANG} ${OPTS} ${CLANG_FLAGS} -c bpf_lb.c -o - | ${LLC} ${LLC_FLAGS} -o /dev/null;)
+	${CLANG} ${CLANG_FLAGS} -c $< -o - | ${LLC} ${LLC_FLAGS} -o $@
 
 LXC_OPTIONS = \
-	 -DUNKNOWN \
```

Comment:
The `-DUNKNOWN` ensured that we compile with no added options. Maybe rename it as it is not obvious.

Reply:
This series added another target which does this as well, a few lines down. That should be sufficient right? Otherwise we would compile with no added options twice.